### PR TITLE
Add debian packaging

### PR DESCRIPTION
**Pull request by rram**
I'm not terribly pleased with the rake/makefile mess and am happy to hear ideas for improvement/cleanliness.

👓 @ckwang8128 @qbrian @spladug

### CODE REVIEW

File: fluent-plugin-logdna.gemspec [hunk 1]
```diff
@@ -3,7 +3,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name        = 'fluent-plugin-logdna'
-  s.version     = '0.1.7'
+  s.version     = '0.1.8'
```
Comment (spladug) [inline fluent-plugin-logdna.gemspec]: i think we want to do 0.1.7.1? i'm worried that this packaging will conflict with whenever upstream releases 0.1.8

**Review by spladug — APPROVED** [review]
with one nit. seems pretty reasonable to me!

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+$(DEBFILE): $(GEMFILE) $(CACHE_DIR)
+	$(BUNDLE) exec $(RAKE) gemdep:minimal
+	$(FPM) --input-type gem --output-type deb \
+	  --depends 'td-agent > 2' \
+	  --no-auto-depends \
+	  --gem-gem $(GEM) \
+	  --no-gem-fix-name \
+	  --deb-build-depends 'td-agent > 2' \
+	  pkg/*.gem $(GEMFILE)
```
Comment (ckwang8128) [inline Makefile]: Are there any non-standard flags here for FPM-ing Ruby stuff? If so, could you add a comment about why the flags are needed?

**Review by ckwang8128 — CHANGES_REQUESTED** [review]
I know we went over most of this in person, but it'd be great if you could add a README or something additional in the Make/Rakefile comments about the hoops that you had to jump through to generate this. 

Just this and my other comment about adding some docs. The structure itself looks okay to me!

**Review by ckwang8128 — APPROVED** [review]

**Pull request merged**